src/utils/normalize-mime-parts.ts

```diff
@@ -73,12 +73,17 @@ export function normalizeMimeParts(
 	message: { [key: string]: any },
 	{ origin, jwtToken }: { jwtToken?: string; origin?: string }
 ) {
-	const processAttachment = ({ ...attachment }) => {
+	const processAttachment = ({ ...attachment }, type: string) => {
 		attachment.messageId = attachment.messageId || message.id;
 		attachment.url = getAttachmentUrl(attachment, { origin, jwtToken });
-		if (attachment.contentId) {
-			attachment.contentId = normalizeCid(attachment.contentId);
-		}
+		attachment.contentId = attachment.contentId
```

Comment:
Use grouping of operation, otherwise it will lead to ambiguity.

Reply:
eslint removes the grouping.